tensorflow/compiler/xla/service/elemental_ir_emitter.cc

```diff
@@ -1905,8 +1905,8 @@ StatusOr<llvm::Value*> ElementalIrEmitter::EmitElementalGather(
         gather_index_index_components.push_back(index[i]);
       }
     }
-
-    if (gather_index_index_components.size() !=
+    const int64 gather_index_index_components_size = gather_index_index_components.size();
+    if (gather_index_index_components_size !=
```

Comment:
```suggestion
    const int64 num_components = gather_index_index_components.size();
    if (num_components !=
```

Plus merge on the same line